runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -1355,95 +1355,58 @@ scan_hex_helper(J9JavaVM *javaVM, char **cursor, UDATA *value, const char *argNa
 }
 
 /**
- * Wrapper for scan_udata_helper, that provides parsing for memory sizes.
- * User should be able to specify the size in GiBs, MiBs, or KiBs (with G,g,M,m,K,k suffixes) or
- * in bytes (no suffix)
+ * Wrapper for scan_udata_memory_size, that provides readable error messages.
+ * @param cursor address of the pointer to the string to parse for the udata.
+ * @param value address of the storage for the udata to be read.
+ * @param argName string containing the argument name to be used in error reporting.
  * @return true if parsing was successful, false otherwise.
  */
 bool
 scan_udata_memory_size_helper(J9JavaVM *javaVM, char **cursor, UDATA *value, const char *argName)
 {
 	PORT_ACCESS_FROM_JAVAVM(javaVM);
+	U_64 result = scan_udata_memory_size(cursor, value);
```

Comment:
type of `result` should be `uintptr_t` (`UDATA` equivalent)

Reply:
Thanks for catching. Opting for UDATA instead here.
See: https://github.com/AlenBadel/openj9/commit/ccfc1d2f8f78e4db042b493e3164912a86fdcc7b